srcpkgs/simavr/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'simavr'
+pkgname=simavr
+version=1.6
+revision=1
+build_style=gnu-makefile
+make_build_args="RELEASE=1"
+make_build_target="build-simavr build-parts"
+make_use_env=1
+hostmakedepends="pkg-config"
+makedepends="elfutils-devel libfreeglut-devel"
+short_desc="A lean, mean and hackable AVR simulator"
+maintainer="Martijn van Buul <martijn.van.buul@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/buserror/${pkgname}"
+distfiles="https://github.com/buserror/${pkgname}/archive/v${version}.tar.gz"
+checksum=a55ad04d055eef5656c49f78bc089968b059c6efb6a831618b8d7e67a840936d
+
+CFLAGS+="-fPIC"
+do_install() {
+	make DESTDIR=${DESTDIR}/usr PREFIX=/usr RELEASE=1 install
+
+	# Package installs directory containing a single headerfile which is
+	# intended to be used by avr-gcc for AVR-targets into the host's
+	# include directory.  Move it to /usr/avr/include instead. This also
+	# prevents it from being moved into the 'devel' subpackage, which is a
+	# good thing as it is required at runtime.
+
+	mkdir -p ${DESTDIR}/usr/avr/include
+	mv ${DESTDIR}/usr/include/simavr/avr ${DESTDIR}/usr/avr/include/simavr
+
+	# Also remove the associated pkgconfig file
+	rm ${DESTDIR}/usr/lib/pkgconfig/simavr-avr.pc
+}
+
+# simavr-devel is a subpackage
+simavr-devel_package() {
+	short_desc+= " - development files"
```

Comment:
Not correctly formatted. Nobody noticed this error?
https://travis-ci.org/void-linux/void-packages/jobs/398756017#L1052